FreeRTOS/Demo/RISC-V-Qemu-sifive_e-FreedomStudio/freedom-e-sdk/env/encoding.h

```diff
@@ -170,10 +170,17 @@
   asm volatile ("csrr %0, " #reg : "=r"(__tmp)); \
   __tmp; })
 
+/* TODO: */

```

Comment:
Does this need to be resolved before merging?

Reply:
I'll remove the TODO and the commented code. Jesse's fix should be OK. It matches the macro definitions in recent versions of `encoding.h` in `riscv-opcodes`.